frontend/cypress/plugins/index.ts

```diff
@@ -11,12 +11,33 @@
 
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
+const axios = require('axios');
 
 /**
  * @type {Cypress.PluginConfig}
  */
 // eslint-disable-next-line no-unused-vars
 module.exports = (on, config) => {
-  // `on` is used to hook into various events Cypress emits
-  // `config` is the resolved Cypress config
-}
+  config.env.cookie = false;
+  // This name is non-standard and might change based on your environment hence the separate
+  // env variable.
+  config.env.AUTH_HTTP_PROVIDER_NAME = config.env.AUTH_HTTP_PROVIDER_NAME || 'my_htpasswd_provider';
```

Comment:
This var name has "HTTP" in it... this doesn't match the doc ... see the [new line 18 of the readme ](https://github.com/kiali/kiali/pull/5270/files#diff-5544425694e0ea24a620263de8dda1c76e42d213916683ff35788135761b3922R18):
```
export CYPRESS_AUTH_PROVIDER_NAME=<value> # defaults to my_htpasswd_provider. Can pass a custom name for the htp provider.
                                          # or optionally openshift for AWS
```

Also - this isn't "HTTP" related.. its "HTPASSWD"

Reply:
Whoops I think my brain must've mixed the two up while typing this. I'll update to be `HTP`.